docs/guide/learning/learning-platforms.md

```diff
@@ -30,26 +30,48 @@ The videos explain in detail real world examples, reusable patterns, no generic
 The instructor and author of the platform, Jeffrey Way, is a pro when it comes to teaching. He has years of experience behind his back, and the quality of the videos is very good.
 
 ### Scotch.io
-[Skotch.io](http://scotch.io/) 
+[Skotch.io](http://scotch.io/) is an online learning platform specifically for web development that covers both videos and writing format.
 
-    A few words about the platform.
-    
-    A sentence or two about their Vue articles and who the platform is good for. 
-    
-    Finishing sentence about instructors, quality of videos, so on.
+Their articles as well as the two video courses they have for Vue goes in depth
 
+The instructors are knowledgeable and have years of experience behind them.
 
 ### Frontend Masters
-[Frontend Masters](https://frontendmasters.com/)
+[Frontend Masters](https://frontendmasters.com/) is an online video  learning platform about JavaScript and front-end development skills such as React, Node.js, Vue, Functional JavaScript, CSS Grid, Flexbox and more!
+
+Their videos are top notched and goes in-depth in the subject topic.
+
+Their instructors are industry leaders and even the creator of Vue has a course also. Definitely worth checking out!
 
 ### Egghead.io
-[Egghead.io](https://egghead.io/)
+[Egghead.io](https://egghead.io/) is a group of working web development professionals and open source contributors that provide you with concise, information dense video courses on the best tools in the industry.
+
+Their videos are straight to the point and deliver knowledge that you can use today. You won't find boring 8 hour courses on egghead, instead you'll find a wealth of bite-sized courses that pack more information in a fraction of the time.
+
+Egghead instructors are industry veterans with years of practical,
+from-the-trenches experience building real software at all scales.
 
 ### Codecourse
-[Codecourse](https://codecourse.com)
+[Codecourse](https://codecourse.com) is a learning platform whose mission is to help you to learn to code with relaxed, easy to follow tutorials.
+
+Their videos are great and focuses on practical projects to get you ahead. Rather than just teach theory, they are focused on real code to inspire your next project. For Laravel Developers that want to learn Vue, this is a great place to start!
```

Comment:
```suggestion
Their videos are great and focus on practical projects to get you ahead. Rather than just teach theory, they are focused on real code to inspire your next project. For Laravel Developers that want to learn Vue, this is a great place to start! 
```